api/v1/openapi.yaml

```
@@ -1736,6 +1736,9 @@ definitions:
       host-firewall:
         description: Status of the host firewall
         "$ref": "#/definitions/HostFirewall"
+      SRv6:
+        description: Status of srv6
```

This one should be in the proper case as it's part of a sentence.
```suggestion
        description: Status of the SRv6 support
```